frontend/src/Connection.ts

```diff
@@ -48,11 +38,15 @@ export class Connection {
     appUpdate: Update,
     disableNodes?: boolean
   ): Promise<Connection> {
+    const rpcUrl = 'wss://eu-0.gemini-2a.subspace.network/ws';
```

Comment:
in a future PR we should probably extract this out and make it configurable

Reply:
yes, I will create an issue for this